crates/core/src/db/update.rs

```diff
@@ -257,21 +257,21 @@ mod tests {
 
     #[test]
     fn test_updates_new_table() -> anyhow::Result<()> {
-        let current = vec![Cow::Owned(TableSchema {
-            table_id: TableId(42),
-            table_name: "Person".into(),
-            columns: vec![ColumnSchema {
+        let current = vec![Cow::Owned(TableSchema::new(
```

Comment:
Let's keep this PR as minimal as possible. Please remove the changes related to this.